ch32v003fun/ch32v003fun.c

```diff
@@ -1119,121 +1129,329 @@ void InterruptVectorDefault()
-	.word   FLASH_IRQHandler           /* 20: Flash */ \n\
-	.word   RCC_IRQHandler             /* 21: RCC */ \n\
-	.word   EXTI0_IRQHandler           /* 22: EXTI Line 0 */ \n\
-	.word   EXTI1_IRQHandler           /* 23: EXTI Line 1 */ \n\
-	.word   EXTI2_IRQHandler           /* 24: EXTI Line 2 */ \n\
-	.word   EXTI3_IRQHandler           /* 25: EXTI Line 3 */ \n\
-	.word   EXTI4_IRQHandler           /* 26: EXTI Line 4 */ \n\
-	.word   DMA1_Channel1_IRQHandler   /* 27: DMA1 Channel 1 */ \n\
-	.word   DMA1_Channel2_IRQHandler   /* 28: DMA1 Channel 2 */ \n\
-	.word   DMA1_Channel3_IRQHandler   /* 29: DMA1 Channel 3 */ \n\
-	.word   DMA1_Channel4_IRQHandler   /* 30: DMA1 Channel 4 */ \n\
-	.word   DMA1_Channel5_IRQHandler   /* 31: DMA1 Channel 5 */ \n\
-	.word   DMA1_Channel6_IRQHandler   /* 32: DMA1 Channel 6 */ \n\
-	.word   DMA1_Channel7_IRQHandler   /* 33: DMA1 Channel 7 */ \n\
-	.word   ADC1_2_IRQHandler          /* 34: ADC1_2 */ \n"
-#if defined(CH32V20x) || defined(CH32V30x)
-"	.word   USB_HP_CAN1_TX_IRQHandler  /* 35: USB HP and CAN1 TX */ \n\
-	.word   USB_LP_CAN1_RX0_IRQHandler /* 36: USB LP and CAN1RX0 */ \n\
-	.word   CAN1_RX1_IRQHandler        /* 37: CAN1 RX1 */ \n\
-	.word   CAN1_SCE_IRQHandler        /* 38: CAN1 SCE */ \n"
-#else
-"	.word   0 \n\
-	.word   0 \n\
-	.word   0 \n\
-	.word   0 \n"
-#endif
-"	.word   EXTI9_5_IRQHandler         /* 39: EXTI Line 9..5 */ \n\
-	.word   TIM1_BRK_IRQHandler        /* 40: TIM1 Break */ \n\
-	.word   TIM1_UP_IRQHandler         /* 41: TIM1 Update */ \n\
-	.word   TIM1_TRG_COM_IRQHandler    /* 42: TIM1 Trigger and Commutation */ \n\
-	.word   TIM1_CC_IRQHandler         /* 43: TIM1 Capture Compare */ \n\
-	.word   TIM2_IRQHandler            /* 44: TIM2 */ \n\
-	.word   TIM3_IRQHandler            /* 45: TIM3 */ \n\
-	.word   TIM4_IRQHandler            /* 46: TIM4 */ \n\
-	.word   I2C1_EV_IRQHandler         /* 47: I2C1 Event */ \n\
-	.word   I2C1_ER_IRQHandler         /* 48: I2C1 Error */ \n\
-	.word   I2C2_EV_IRQHandler         /* 49: I2C2 Event */ \n\
-	.word   I2C2_ER_IRQHandler         /* 50: I2C2 Error */ \n\
-	.word   SPI1_IRQHandler            /* 51: SPI1 */ \n\
-	.word   SPI2_IRQHandler            /* 52: SPI2 */ \n\
-	.word   USART1_IRQHandler          /* 53: USART1 */ \n\
-	.word   USART2_IRQHandler          /* 54: USART2 */ \n\
-	.word   USART3_IRQHandler          /* 55: USART3 */ \n\
-	.word   EXTI15_10_IRQHandler       /* 56: EXTI Line 15..10 */ \n\
-	.word   RTCAlarm_IRQHandler        /* 57: RTC Alarm through EXTI Line */ \n"
-#if defined(CH32V10x) || defined(CH32V20x) || defined(CH32V30x)
-"	.word   USBWakeUp_IRQHandler       /* 58: USB Wake up from suspend */ \n"
-#if defined(CH32V20x) || defined(CH32V30x)
 
-#if defined(CH32V30x)
-"   .word   TIM8_BRK_IRQHandler        /* 59: TIM8 Break == masked to USBHD? */ \n"
-#else
-"	.word   USBHD_IRQHandler           /* 59: USBHD Break */ \n"
+#if defined(CH32V10x)
+"	.word   SysTick_Handler            /* SysTick Handler */ \n \
+    	.word   0 \n \
```

Comment:
Removing the additional tab indent here should fix some of the unnecessary diff changes.